### ISSUE TITLE
[IDEA] /stat.ps3 icon/bookmarklet in ps3 internet broswer

### ISSUE THREAD

**Issue by retrosapien** [aldostools/webMAN-MOD#481]
[EDIT]
The stat command does not seem to be working as expected. How do you use it recursively? Wild card after the / doesn't seem to work. 
I thought I remembered it working correctly before, or has it always been like this?

Using the context menu(right click) in pc browser i can get the folder size. Is there an equivalent to do this in the native internet browser on the ps3.
 I figured out how to manually use the stat.ps3 command in the url by going to the directory i want and then pressing start and adding the command to the url. 

An onscreen option could still be helpful to new users, though.

I was wondering if it is possible to do this through a bookmarklet, or if bookmarklets are even possible on the ps3 internet browser? Does it even have javascript?

Anyway, like i mentioned, there is a workaround, so not a priority.

Thanks for everything you have done for the ps3 scene.

**Comment by aldostools** [comment]
Thank you for the feedback. The /stat.ps3 suggestion will be included in the next update.

**Comment by retrosapien** [comment]
Thank you for considering it. I think it will be a very useful addition.

Can I get some clarification on whether it is supposed to stat recursively by default? Or how to do that if it is not?

Thank you,

Tim (retrosapien)



Sent from my Samsung Galaxy smartphone.


-------- Original message --------

**Comment by retrosapien** [comment]
@aldostools 
Implementing this idea only requires editing the html file, right?
If that's all it is I might try it out myself. 
If it's any of the other Web files, that's probably beyond my current capabilities. =)

**Comment by aldostools** [comment]
No, the link is generated by the plugin including the current path.
The /stat.ps3 is implemented in the latest build. Download the latest installer. 

The link also could be generated adding custom javascript to the html templates, but you need to have development skills in that language to do it.

**Comment by perna1986** [comment]
V
